apps/system/js/ftu_ping.js

```diff
@@ -358,9 +339,13 @@
           this.debug('No OS information, holding off');
           return false;
         }
-
-        this.ping();
-        return true;
```

Comment:
Hi Marshall,
So this boolean return value is just for unit testing and has no bearing on the functionality.  What finally gets clearInterval called is when ping() is called.  I had to remove it because it only returns true when it gets to the end of the function.  I could not return the true after the promise was resolved and ping was called as it would have chained me into the next clause in the promise.